test.py

```diff
@@ -1306,6 +1309,11 @@ def read_log(log_filename: pathlib.Path) -> str:
 
 
 def print_summary(failed_tests, options: argparse.Namespace) -> None:
+    rusage = resource.getrusage(resource.RUSAGE_CHILDREN)
+    cpu_used = rusage.ru_stime + rusage.ru_stime
```

Comment:
should this be ru_stime + ru_utime instead?

Reply:
yes, fixed